test/from-draft3/to-draft4/divisiblyBy-has-value-less-than-zero.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "title": "`divisibleBy` has value less then 0",
+        "from": {
+            "divisibleBy": -1
+        },
+        "to": {
+            "divisibleBy": 1
```

Comment:
Shouldn't this be `multipleOf`?

Reply:
😅 this will also be a preprocessing rule, the immediate next rules after this will make move operation from divisibleBy to multipleOf.
OR do you suggest that we should avoid the inter dependencies between rules as much as possible? and only have ordering dependency where there is no option...
-- anyway I think lets keep it now and at the end we will remove inter dependencies...?

Reply:
Interdependencies between rules is OK. I think we should optimise for having many simple rules powered with simple conditions & transforms DSLs rather than big interdependent rules that are very complex

Reply:
But for example in this case, there should be a rule that renames `divisibleBy` to `multipleOf`?

Reply:
@jviotti  I have merged both into a single rule because it doesn't break and covers all the cases..